test/studies/diffusion/heat_2d.chpl

```diff
@@ -1,35 +1,45 @@
-// setup simulation parameters
-config const xLen : real = 2,
-             yLen : real = 2,
-             nx = 31,
-             ny = 31,
-             nt = 50,
-             sigma = 0.25,
-             nu = 0.05;
-
-const dx : real = xLen / (nx - 1),
-      dy : real = yLen / (ny - 1),
-      dt : real = sigma * dx * dy / nu;
-
-// define a 2D domain and subdomain
-const dom = {0..<nx, 0..<ny},
-      domInner : subdomain(dom) = dom.expand(-1);
-
-// define an array to hold the solution
-var u : [dom] real;
-
-// setup initial conditions
-u = 1;
+// ---- set up simulation parameters ------
+// declare configurable parameters with default values
+config const xLen = 2.0,    // length of the domain in x
+             yLen = 2.0,    // length of the domain in y
+             nx = 31,       // number of grid points in x
+             ny = 31,       // number of grid points in y
+             nt = 50,       // number of time steps
+             sigma = 0.25,  // CFL condition
+             nu = 0.05;     // viscosity
+
+// declare non-configurable parameters
+const dx : real = xLen / (nx - 1),       // grid spacing in x
+      dy : real = yLen / (ny - 1),       // grid spacing in y
+      dt : real = sigma * dx * dy / nu;  // time step size
```

Comment:
> Just to make sure this was by intention rather than oversight, I assume you decided you preferred keeping the : real type declarations here even though they're not strictly necessary?

Yeah, I forgot to make a note of it on the other PR, but I would like to keep these for extra clarity. I will update the spacing though.

Thanks for all the feedback!